pkgs/top-level/all-packages.nix

```diff
@@ -37367,4 +37367,6 @@ with pkgs;
   swaysettings = callPackage ../applications/misc/swaysettings { };
 
   aitrack = libsForQt5.callPackage ../applications/misc/aitrack { };
+
+  owofetch = callPackage ../tools/misc/owofetch { };
```

Comment:
Generally this list is alphabetical, not sure why it falls apart at the bottom :thinking: 

This should be moved under `owamp = callPackage ../applications/networking/owamp { };`

Reply:
I was trying to decide if it was or not, and as noted it fell apart so i just assumed it was chronological.